ci-bench/src/rustls_facade.rs

```diff
@@ -0,0 +1,60 @@
+use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
+use rustls::{ConnectionCommon, SideData};
+use std::io::{Read, Write};
+
+/// Sends data to the other side of the connection through the provided writer
```

Comment:
editorial nit: missing `.`
```suggestion
/// Sends data to the other side of the connection through the provided writer.
```